src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1609,6 +1609,7 @@ public function getPreferredFormat(?string $default = 'html'): ?string
     public function getPreferredLanguage(?array $locales = null): ?string
     {
         $preferredLanguages = $this->getLanguages();
+        $locales = array_map($this->formatLocale(...), $locales ?? []);
```

Comment:
Should be moved line 1621 maybe ? (micro-picking :) )

Reply:
Indeed. It moved this line